cfg/builder/builder_finalize.cc

```diff
@@ -288,13 +299,12 @@ void CFGBuilder::computeMinMaxLoops(core::Context ctx, const CFG::ReadsAndWrites
             continue;
         }
 
-        for (const core::LocalVariable what : RnW.reads[bb->id]) {
-            const auto minIt = cfg.minLoops.find(what);
-            int curMin = minIt != cfg.minLoops.end() ? minIt->second : INT_MAX;
```

Comment:
Note: `minLoops` is now initialized to `INT_MAX` when we resize it to size `maxVariableId`.